comps/navbar.js

```diff
@@ -1,9 +1,26 @@
 import Link from 'next/link'
+import styles from '../styles/Home.module.css'
+import {useState} from 'react'
 
-function Navbar(props){
+function Navbar({q, currentQ}){
+
+	const qID = currentQ + 1
```

Comment:
try not to use magic numbers, but instead use constants for those. Also extract this to a function